### ISSUE TITLE
Optimizer文档示例代码问题（版本号2.1）

### ISSUE THREAD

**Issue by chenqianhe** [PaddlePaddle/Paddle#36670]
## 示例代码
![Snipaste_2021-10-24_12-05-26](https://user-images.githubusercontent.com/54462604/138580374-520ed6d8-53e0-4828-84d1-2aa936715a3c.png)
## 问题
Optimizer示例代码中有
```python
out.backward()
```
是否应为
```python
loss.backward()
```
## 文档链接
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/Optimizer_cn.html#optimizer

**Comment by cuicheng01** [comment]
这里确实应该用`loss.backward()`，感谢反馈，我们会尽快修复～
